.github/workflows/test.yml

```diff
@@ -17,6 +17,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Build and test
+      - name: Build and test Catbird package
         run: make test
         shell: bash
+      - name: Build and test CatbirdApp package
+        run: make test
+        shell: bash
+        working-directory: Packages/CatbirdApp
```

Comment:
I think in test.xml you can run tests without a Makefile so that they are readable. Or call make test from the root and have it run everything you need inside

Reply:
I've made single Makefile.